README.md

```diff
@@ -48,21 +48,31 @@ Note that Vroom requires a version of vim built with the `+clientserver`
 option (run `vim --version` to check).  See `:help clientserver` for
 additional requirements.
 
-The easiest way to install vroom is to cd into the vroom directory and run
+If you're on Ubuntu or Debian, you can install
+[release packages](https://github.com/google/vroom/releases) from GitHub.
+
+Otherwise, the easiest way to install vroom is to clone the vroom repository
+from Github, cd into the vroom directory, and run
 ```sh
 python3 setup.py build && sudo python3 setup.py install
 ```
 
-You can also install
-[release packages](https://github.com/google/vroom/releases) from github. To
-build deb packages yourself, make sure you have a recent stdeb version with
-python 3 support installed.
-
 Vim 7.4.384 and later have built-in syntax support for the vroom filetype. You
 can install the standalone
 [ft-vroom plugin](https://github.com/google/vim-ft-vroom) for older versions of
 vim.
 
+## Usage
+
+Vroom is invoked from the command-line on `.vroom` files.
```

Comment:
It is, but without a little more explanation of what `vroom somefile.vroom` actually does, I feel like mentioning `--crawl` would be diluting the basic explanation of "How to use vroom". I'll probably expand it with a richer explanation soonish (or +1 if you want to).